browser/src/slideshow/LayerDrawing.ts

```diff
@@ -265,6 +266,24 @@ class LayerDrawing {
 		VideoRendererGl.deleteProgram(this.layerRenderer.getRenderContext());
 	}
 
+	private drawNavigationButtons(slideHash: string) {
+		console.log('LayerDrawing.drawNavigationButtons: slideHash: ' + slideHash);
+		const slideInfo = this.getSlideInfo(slideHash);
+		if (slideInfo.next) {
+			console.log('slideInfo.next');
+			var image = new Image();
+			image.onload = () => {
+				console.log('image loaded');
+				createImageBitmap(image).then((imageInfo) => {
+					console.log('image created');
+					console.log(imageInfo);
+					this.drawBitmap(imageInfo);
+				});
+			};
+			image.src = 'http://localhost:9980/browser/99d3c9c049/images/lc_next.svg';
```

Comment:
You can use a css class, or a relative path "images/lc_next.svg" directly.

Reply:
best to use `LOUtil.setImage`: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.JSDialogBuilder.js#L617

which already supports theming, etc.

Reply:
but there are similar helpers for getting just icon URL (if theming is not needed).